lineage/figures/figure12.py

```diff
@@ -0,0 +1,213 @@
+                                         scale=scale1[9],
+                                         size=500),
+                            sp.gamma.rvs(a=a0[8], loc=gamma_loc,
+                                         scale=scale0[8],
+                                         size=500),
+                            sp.gamma.rvs(a=a1[8],
+                                         loc=gamma_loc, scale=scale1[8],
+                                         size=500),
+                            sp.gamma.rvs(a=a0[7], loc=gamma_loc,
+                                         scale=scale0[7], size=500),
+                            sp.gamma.rvs(a=a1[7], loc=gamma_loc,
+                                         scale=scale1[7], size=500),
+                            sp.gamma.rvs(a=a0[6], loc=gamma_loc,
+                                         scale=scale0[6], size=500),
+                            sp.gamma.rvs(a=a1[6], loc=gamma_loc,
+                                         scale=scale1[6], size=500),
+                            sp.gamma.rvs(a=a0[5], loc=gamma_loc,
+                                         scale=scale0[5], size=500),
+                            sp.gamma.rvs(a=a1[5], loc=gamma_loc,
+                                         scale=scale1[5], size=500),
+                            sp.gamma.rvs(a=a0[4], loc=gamma_loc,
+                                         scale=scale0[4], size=500),
+                            sp.gamma.rvs(a=a1[4], loc=gamma_loc,
+                                         scale=scale1[4], size=500),
+                            sp.gamma.rvs(a=a0[3], loc=gamma_loc,
+                                         scale=scale0[3], size=500),
+                            sp.gamma.rvs(a=a1[3], loc=gamma_loc,
+                                         scale=scale1[3], size=500),
+                            sp.gamma.rvs(a=a0[2],
+                                         loc=gamma_loc, scale=scale0[2],
+                                         size=500),
+                            sp.gamma.rvs(a=a1[2], loc=gamma_loc,
+                                         scale=scale1[2], size=500),
+                            sp.gamma.rvs(a=a0[1], loc=gamma_loc,
+                                         scale=scale0[1], size=500),
+                            sp.gamma.rvs(a=a1[1], loc=gamma_loc, 
+                                         scale=scale1[1], size=500),
+                            sp.gamma.rvs(a=a0[0], loc=gamma_loc,
+                                         scale=scale0[0], size=500),
+                            sp.gamma.rvs(a=a1[0], loc=gamma_loc,
+                                         scale=scale1[0], size=500)))
+
+    dists = pd.DataFrame(columns=['lifetime [hr]', 'distributions', 'hues'])
+    dists['lifetime [hr]'] = dist1
+    dists['distributions'] = 1000 * (['d1']) + 1000 * (['d2']) + \
+        1000 * (['d3']) + 1000 * (['d4']) + 1000 * (['d5']) + \
```

Comment:
You can loop and append these to the DataFrame in each iteration. I think that'd be cleaner.